bpf/lib/policy_log.h

```diff
@@ -62,7 +80,7 @@ send_policy_verdict_notify(struct __ctx_buff *ctx, __u32 remote_label, __u16 dst
 static __always_inline void
 send_policy_verdict_notify(struct __ctx_buff *ctx, __u32 remote_label, __u16 dst_port,
 			   __u8 proto, __u8 dir, __u8 is_ipv6, int verdict,
-			   __u8 match_type)
+			   __u8 match_type, __u32 filter)
```

Comment:
Looks like this is from a previous version.

Reply:
Thanks for the catch! Interesting that compiler didn't catch this error.

Reply:
Yeah, that was weird to me too. We compile test with different macros in [`bpf/Makefile`](https://github.com/cilium/cilium/blob/0b5e263ff749c283481d0cc3b1b6e9165292b2c3/bpf/Makefile#L118-L133), but the `POLICY_VERDICT_NOTIFY` macro isn't part of those. It only guards these few lines, so I'm not sure it's worth it after all.

Reply:
FWIW ideally the bpf/Makefile default build will build the most complex program possible, so that we catch complexity issues with particular configurations in CI before users hit them.

If this filter is adding complexity (which it is, since we've had CI issues in that area), then we should ensure that the corresponding option is enabled by default in the `bpf/Makefile`.

Reply:
It's currently enabled by default in `lxc_config.h`; that should be enough, right?

Reply:
I guess that CI is already catching these, but I was thinking of `test/bpf/check-complexity.sh` which loads the version built with the options from the Makefile. Would be good to fix this up either in this PR or as an immediate follow-up PR.

Reply:
Ah, right. I wasn't thinking about `check-complexity.sh`. I can take care of that in a follow-up PR; I suspect `POLICY_VERDICT_NOTIFY` isn't the only missing option.

Reply:
I looked again at this PR and in this case and it's already achieved, no further changes are necessary.